src/geometry/manifold/times_cont_mdiff.lean

```diff
@@ -249,13 +249,41 @@ begin
   mfld_set_tac
 end
 
+/-- One can reformulate smoothness within a set at a point as continuity within this set at this
+point, and smoothness in the corresponding extended chart in the target. -/
+lemma times_cont_mdiff_within_at_iff_target :
+  times_cont_mdiff_within_at I I' n f s x ↔ continuous_within_at f s x ∧
+    times_cont_mdiff_within_at I (model_with_corners_self 𝕜 E') n ((ext_chart_at I' (f x)) ∘ f)
```

Comment:
There is the localized notation `𝓘(𝕜, E)` for `model_with_corners_self 𝕜 E` in `manifold` that helps make things more readable (and that we should probably adopt more widely in mathlib).